### DISCUSSION TITLE
-nan(ind)

### DISCUSSION THREAD

**Discussion by TheSimplyRed** [coin-or/Ipopt#564] [Q&A] [answered]
Hi,
I got something strange. During some iterations I have x values that becomes -nan(ind)!
What can be wrong in such cases?

**Comment by svigerske** [comment, marked as answer]
Some computations lead to `nan`.

First try turning on option [check_derivatives_for_naninf](https://coin-or.github.io/Ipopt/OPTIONS.html#OPT_check_derivatives_for_naninf) and make sure you use Ipopt 3.14.

If that doesn't help, compile Ipopt with `--enable-debug --with-ipopt-checklevel=2` (https://coin-or.github.io/Ipopt/INSTALL.html#CONFIGURE_FLAGS) and/or set a high `print_level` to maybe see which operations cause the `nan`'s.
